test/format/format.i

```diff
@@ -34,27 +34,45 @@ static inline int
 read_op(WT_CURSOR *cursor, read_operation op, int *exactp)
 {
     WT_DECL_RET;
+    uint64_t start, now;
 
     /*
      * Read operations wait out prepare-conflicts. (As part of the snapshot isolation checks, we
      * repeat reads that succeeded before, they should be repeatable.)
      */
+    __wt_seconds(NULL, &start);
     switch (op) {
     case NEXT:
-        while ((ret = cursor->next(cursor)) == WT_PREPARE_CONFLICT)
+        while ((ret = cursor->next(cursor)) == WT_PREPARE_CONFLICT) {
+            __wt_seconds(NULL, &now);
+            testutil_assertfmt(now < start || now - start < 60,
```

Comment:
I'm not sure how `now` would ever be < `start` here. Are you defensively coding against a clock reset? Having that as the first part of the assertion makes it seem like the expected case. Maybe:
`testutil_assertfmt(now >= start && now - start < 60);`
For some reason that makes a lot more sense in my head.

Reply:
I just want to ignore it, it should never happen (but I don't want to chase the failure, either). I added a comment in d3a066134.